lib/src/model/board_editor/board_editor_controller.dart

```diff
@@ -59,14 +61,66 @@ class BoardEditorController extends _$BoardEditorController {
     state = state.copyWith(
       sideToPlay: side,
     );
+    _calculateEnPassantOptions();
```

Comment:
It is better to have atomic state changes, so:
```dart
state = state.copyWith(
  sideToPlay: side,
  enPassantOptions: _calculateEnPassantOptions(),
);
```

Reply:
Done here: https://github.com/lichess-org/mobile/commit/9dc7df076e8e1eb9a5b1e03204e5dfcd8b88de74